02_bubble_sort_anishaNair/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(n []int) []int {
+	if len(n) == 0 {
```

Comment:
Do we need this? If n is empty string, it will be returned.

Reply:
removed the condition check